pkg/block/adapter/errors.go

```diff
@@ -0,0 +1,5 @@
+package adapter
+
+import "github.com/pkg/errors"
+
+var ErrNotFound = errors.New("not found")
```

Comment:
This is a somewhat confusing name: AFAICT it means "gone", so perhaps
```suggestion
var ErrGone = errors.New("data not found")
```
or
```suggestion
var ErrDataNotFound = errors.New("data not found")
```